Dockerfile

```diff
@@ -1,34 +1,48 @@
-FROM node@sha256:dbb1e74774617c46a34f67d25decbdc5841af2a2a08a125a37743c734799b909 as builder
-# Docker Tag: node:8.11.1-stretch
+FROM parity/parity:v1.9.7
 
-ADD ./vendor /tmp/vendor
+ADD ./key.json /tmp/keystore/
+ADD ./setup_8.x /tmp/
```

Comment:
Consider moving these to after the apt-get and git-clone stuff.  It is more likely you'll iterate/change local files than change the apt-get lines, also the apt-get lines are _way_ more time consuming than file adds.  This way, you'll get better layer caching results.

Reply:
Ah, I just noticed that setup is executed _before the apt-get stuff.  Can we execute setup after apt-get?